backend/requirements-test.txt

```diff
@@ -17,3 +16,15 @@ testing.postgresql>=1.3.0
 flask-shell-ipython
 python-dotenv
 watchdog
+
+# All requirements for oio_rest
+-e git+https://github.com/magenta-aps/mox@development#egg=oio-rest&subdirectory=oio_rest
+pytz==2018.5
+Jinja2==2.10
```

Comment:
`mora` already depends on some of these packages — at the very least `Jinja2` & `Werkzeug` — and we should purge the dependencies to avoid duplicates.